### ISSUE TITLE
blocking-http-call-in-async-function (ASYNC210) false negative on `requests.request`

### ISSUE THREAD

**Issue by jelle-openai** [astral-sh/ruff#28015]
### Summary

ASYNC210 and S113 both flag `requests` functions in some contexts (respectively, use in async functions and use without a timeout), but differ in some cases:
```python
import requests

async def f(url: str):
    requests.get(url) # ASYNC210, S113
    requests.request("GET", url) # S113 only
```

These lints both look at similar issues, but only the timeout one (S113). This also affects `httpx.request`.

### Version

0.16.4
